portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -29,9 +29,9 @@
 @WebServlet("/login-status")
 public class LoginStatusServlet extends HttpServlet {
   
-  private static final Gson gson = new Gson();
-  private static final String urlToRedirectToAfterUserLogsOut = "/";
-  private static final String urlToRedirectToAfterUserLogsIn = "/";
+  private static final Gson GSON = new Gson();
```

Comment:
We don't usually view gson as a constant. It's more like a service so it'd be better to write it as `gson`

Reply:
Makes sense ty!